errai-jaxrs/errai-jaxrs-client/src/test/java/org/jboss/errai/enterprise/jaxrs/server/QueryParamTestServiceImpl.java

```diff
@@ -55,6 +56,11 @@ public String getWithMultipleQueryParams(long id1, long id2) {
     return id;
   }
 
+  @Override
+  public Set<String> getWithQueryParamSetOfStringsTreeSet(Set<String> id) {
+    return new TreeSet<>(id);
+  }
+
```

Comment:
why is this here in alignment PR?

Reply:
brought in by the PR @tiagobento raised to fix failing tests in errai